calculator/main.py

```diff
@@ -37,14 +47,18 @@ def add_new_operation(calc, operation):
     :param operation: Dict with the single operation to be added.
                       ie: {'add': add_function}
     """
-    pass
-
+    # validate operation is a dictionary
```

Comment:
Since this is a class, I like that you commented your code because it'll be good to look back at in the future.